docs/package.json

```diff
@@ -26,5 +26,8 @@
     "react-typography": "^0.16.18",
     "typography": "^0.16.18",
     "typography-theme-github": "^0.16.19"
+  },
+  "dependencies": {
+    "gatsby-source-git": "^1.0.2"
```

Comment:
I've only used `devDependencies`, as I don't think any gatsby dependency is needed in production?

Reply:
Ah that's just my habit of installing all my Gatsby things as dependencies. I just moved it over to `devDependencies` 👍